### PR TITLE
Fix issues on scanning tools

### PR DESCRIPTION
**Pull request by starwarfan**

### CODE REVIEW

**Comment by starwarfan** [comment]
@HuaChunbo Try this on checkmarx to see if it works

File: source/agent/sip/sipIn/sip_gateway/sipua/src/sipua.c [hunk 1]
```diff
@@ -55,7 +55,7 @@ static int construct_uag(struct uag **uagp, void *ep, const char *sip_server, co
 	int err;
 	char account_str[512]={0};
 
-    sprintf(account_str, "%s <sip:%s:%s@%s>\n", disp_name, user_name, password, sip_server);
+    sprintf(account_str, "%.100s <sip:%.100s:%.100s@%.100s>\n", disp_name, user_name, password, sip_server);
```
Comment (xiandeatintel) [inline source/agent/sip/sipIn/sip_gateway/sipua/src/sipua.c]: "%.100s <sip:%.100s:%.100s@%.100s>\n" ->
"%.128s <sip:%.128s:%.64s@%.128s>\n"

**Comment by starwarfan** [comment]
@xiandeatintel We pass the scanning finally

**Comment by xiandeatintel** [comment]
Please @daijh help review and approve the change upon MsdKFrame module.

**Pull request merged**